### PR TITLE
Call control api

### PR DESCRIPTION
**Pull request by srs-binnys**

### CODE REVIEW

File: LiveVox.NET/LiveVox.NET.csproj [hunk 1]
```diff
@@ -6,6 +6,12 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Models\CallControl\**" />
+    <EmbeddedResource Remove="Models\CallControl\**" />
```
Comment (srs-adamr) [inline LiveVox.NET/LiveVox.NET.csproj]: Check on this

File: LiveVox.NET/Models/Call/Enumerations/ErrorCode.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Call/Enumerations/ErrorCode.cs:1]: This is commented

File: LiveVox.NET/Models/Call/Requests/Agent/PreviewDial/PlayMessageRequest.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Call/Requests/Agent/PreviewDial/PlayMessageRequest.cs:1]: This is in the wrong namespace. Should be in the Message namespace.

**Review by srs-adamr — CHANGES_REQUESTED** [review]
Few things, looks good otherwise!

File: LiveVox.NET/LiveVox.NET.csproj (hunk 1 above)
Comment (srs-binnys) [reply LiveVox.NET/LiveVox.NET.csproj]: Have removed these lines Adam, they were not required.

File: LiveVox.NET/Models/Call/Enumerations/ErrorCode.cs
Comment (srs-binnys) [reply LiveVox.NET/Models/Call/Enumerations/ErrorCode.cs:1]: Uncommented the error codes.

File: LiveVox.NET/Models/Call/Requests/Agent/PreviewDial/PlayMessageRequest.cs
Comment (srs-binnys) [reply LiveVox.NET/Models/Call/Requests/Agent/PreviewDial/PlayMessageRequest.cs:1]: Moved the files to Message Namespace.

File: LiveVox.NET/Models/Call/Enumerations/ErrorCode.cs [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+using System.Threading.Tasks;
+
+namespace LiveVox.NET.Models.Call.Enumerations
+{
+    public enum ErrorCode
+    { 
+        /// <summary>
+        /// Error code 440: Busy
+        /// </summary>
+        [JsonPropertyName("440")]
```
Comment (srs-binnys) [inline LiveVox.NET/Models/Call/Enumerations/ErrorCode.cs:15]: @srs-adamr here we have two properties with the same value Busy, 
the same for OutsideOfAllowableDialingWindow too.

**Review by srs-adamr — APPROVED** [review]

**Review by srs-vimalk — APPROVED** [review]

**Pull request merged**